great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -200,16 +203,12 @@ def _init_domain_builder(
 
         return domain_builder
 
+    @staticmethod
     def _init_parameter_builders(
-        self,
-        rule_config: CommentedMap,
-        data_context: Optional["DataContext"],  # noqa: F821
+        parameter_builder_configs: Optional[List[CommentedMap]] = None,
```

Comment:
```suggestion
        parameter_builder_configs: List[CommentedMap],
```

Reply:
@cdkini Great catch -- of course!  I am addressing it (slightly differently).  Thanks!